bot_modules/attachment_converter/index.js

```diff
@@ -0,0 +1,52 @@
+const Discord = require('discord.js')
+const axios = require('axios')
+const path = require('path')
+
+const { configEditor: config } = require('../config_manager/index.js')
+
+exports.init = (client) => {
+  client.on('message', async (msg) => {
+    msg.attachments.forEach(async (attachment) => {
+      if (config.get().convertExtensions.includes(path.extname(attachment.name))) {
```

Comment:
Maybe check if the file size is below a certain threshold?

Reply:
Showing a preview of the content too would be nice but it' might be something for a separate PR